src/Module/Api/Mastodon/Media.php

```diff
@@ -71,7 +72,15 @@ public function put(array $request = [])
 
 		$photo = Photo::selectFirst(['resource-id'], ['id' => $this->parameters['id'], 'uid' => $uid]);
 		if (empty($photo['resource-id'])) {
-			DI::mstdnError()->RecordNotFound();
```

Comment:
I'm not sure which case we are addressing here. Is this for a case where we were passed in a post-media ID instead of a photo ID so we are updating the description of the post-media table entry instead of the original Photo entry instead?

Reply:
Correct.